packages/location/src/LocationEditor.tsx

```diff
@@ -168,16 +171,18 @@ export function LocationEditorConnected(props: LocationEditorConnectedProps) {
     >
       {({ value, disabled, setValue, externalReset }) => {
         return (
-          <LocationEditor
-            // on external change reset component completely and init with initial value again
-            key={`location-editor-${externalReset}`}
-            value={value}
-            disabled={disabled}
-            setValue={setValue}
-            googleMapsKey={googleMapsKey}
-            selectedView={selectedView}
-            setSelectedView={setSelectedView}
-          />
+          <>
```

Comment:
why do we need a fragment here? :o

Reply:
Have removed, sorry it was left over from testing stuff